src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowLccTest.java

```diff
@@ -122,7 +122,7 @@ private Network createNetwork() {
                 .setConnectableBus("b3")
                 .setBus("b3")
                 .setPowerFactor(0.8f)
-                .setLossFactor(0.1f)
+                .setLossFactor(1.1f)
```

Comment:
Not clear to me this change, in the previous code it was (1 + a) and now it is (1 + a/100). So it was already a deviation but not in % but with a [0, 1] value? Why was the loss factor 0.1.

Reply:
In fact I just changed the value to have a more realistic value. 1.1% of losses during the AC/DC convertion. Before it was 10%...